client/my-sites/exporter/guided-transfer-card/index.jsx

```diff
@@ -111,6 +115,7 @@ class GuidedTransferCard extends Component {
 const mapStateToProps = state => ( {
```

Comment:
Minor but we can store the selectedSiteId in a variable since this is used multiple times.
```javascript
const mapStateToProps = ( state ) => {
   const selectedSiteId = getSelectedSiteId( state );
   return {
       siteId,
       siteSlug: getSiteSlug( state, siteId ),
       //... 
   }
```